### ISSUE TITLE
Add a chapter on fostering a communal office feel while distributed

### ISSUE THREAD

**Issue by quaid** [open-organization/open-org-distributed-work-guide#21]
There is a metaphor for the experience of thriving in an office environment, to consider it like a pond, where the various pebble drops of nuanced communication throughout the workday send ripples only discernible when in person.

I read that metaphor [in an article in 2009](https://randsinrepose.com/archives/the-pond/)--the middle of my career as a distributed team member--and I still can feel the challenge it posed to me. I knew I would likely never work co-located with a team. Would I always have that dis-located feeling, as if my senses were dulled and fuzzy when at remote? So much of that article relied upon intentional management and willing teammates to get it right. Had I just been lucky so far, or was there something we were all doing that leant itself to scaling that pond effect? 

In this chapter, I'll draw on my experience and continued pondering of those questions in the years since. I propose to write about how you take the communication concepts at the core of a co-located pond experience, and scale those via people, processes, and tools into a way to detect the ripples of drops made an ocean away. The method is both directive and organic.

Here's an outline draft, how does this sound?

1. Communication as ripples in the pond -- introduction to the metaphor and how we're going to solve for the problems it presents.
   1. tl;dr of the solution -- find ways to build strong, non-work-focused relationships in a distributed team so that the members amplify the ripples across the ocean to where they need to go.
1. Finding and amplifying parallel pebbles
   1. Allow and encourage the personal touch
       * Using personal and public social media along with organic friendships to create a flow of what is happening in team members' personal, other organizational, creative, etc. lives. Some folks will become friends on Facebook and see one facet of each other; a different one if via Instagram or Snapchat or Twitter. By explicitly encouraging these connections, people see a more complete picture of what is happening in each others' lives at any given moment. And can then share a range of that with the team members who aren't in that direct social media connections. An analogy is if three people in an office all go to the same gym; there is a camaraderie from that, and each of them may share parts of their gym experience with other team members who aren't at that gym. Soon, a shared story of interest to the core three can end up growing beyond
   1. Create and encourage private spaces, both virtual and physical
       * Part of having an open culture work effectively is giving private space for people to share with each other, work out issues, offer and receive advice, be vulnerable with people they trust. When teams meet in person and have time for more social bonding and private discussions, this fosters the ability to have and hold trust when just a voice on the phone the rest of the time. We are able to call the other person to mind, and gain some of the very real physical benefits as if they were in person. It is key that the people, processes, and tools provide trustworthy areas of privacy -- you know in person you can walk away from the group for some 1:1 discussion without being overheard, where can you do that with the distributed team?
1. Recognizing, sharing, and supporting different communication styles
   1. Have everyone on the team provide an honest preference for how they like to communicate for what kind of information.
       * E.g. "Text or group chat; then email; cal invite for necessary meetings; unscheduled phone calls for urgent." Your preference might be to get a phone call for anything timely, and to read email for anything not urgent. I might want to have everything urgent/timely start with a text message or a group chat ping, then we pick the medium to go to from there, schedule and appear. The sight of a group chat ping might set your teeth on edge. The ringing of my phone might surge my adrenalin. So we have to work that out, if we're going to work together. The team should provide the norm and the framework for individuals to see each others' needs and visibly respect those wishes.
1. Creating, building, and energizing relationships
   1. Hold face-to-face meetings with the entire team, where possible.
       * Obvious and usually worth it, but it has a dark side. You may not be able to get the entire team together, and now you have one or more people who are missing a significant high-bandwidth interpersonal time.
   1. Support the crossover of hobbies and interests.
       * You may not be able to have a softball team with everyone from the office, but is there an e-sport you can do to foster that kind of team building through friendly competition? Look for shared hobbies and interests amongst distributed team mates, enabling and encouraging some kind of collaboration. Quilters could share patterns or exchange quilting squares. Video gamers could join the same MMORPG. Chess players can play by (e)mail.
      1. A key part of this is to create and encourage time to talk about these things during and around meetings and group chats. Resist the urge to "get people back on topic" every time, considering that this is there time by the water cooler.
   1. Hold remote team social hours. Play games, watch cartoons together, etc.

I think there may be a few more parts/topics that arise as I get into categorizing the different communication mediums and the interweb between individuals in support of a stronger group.

**Comment by semioticrobotic** [comment]
I really like this, @quaid, and I think it'll work well in the book. I'm thinking we'll be featuring an entire section on "building community," and I'm thinking this might work well as a kind of introductory chapter to that section.

**Comment by quaid** [comment]
Yes, building community sounds like a good container for this. It's interesting to me that I avoided language like that--it gets used A LOT on community architect work, but I tend to label outward community building efforts as such without as easily recognizing the same things work on the other side of the prism.

**Comment by quaid** [comment]
@semioticrobotic  Hey, I dropped out of the editorial loop and my ballpoint pen dried up for a while. Is there still a purpose or use for finishing this content, all or some of it? I'm feeling my ink flowing again with some capacity for this (or something else, but coming back through existing good-ideas-in-progress first). Thanks!

**Comment by semioticrobotic** [comment]
We'd absolutely still love to have it. Whether it lands in v1.0 will depend on your timeline, but either way we'd love to see it in the book.
